### ISSUE TITLE
DMG cannot be run on OS X

### ISSUE THREAD

**Issue by arnounesco** [cstarcher/pysaf#6]
OS X does not allow anymore to run code that has been downloaded and is coming from an unknown developer.
I tried to make a pull request for the changes I made in the README describing how PySAF can be installed manually on OS X but got an access denied.

**Comment by cstarcher** [comment]
@arnounesco 

Yes, I've not updated the README for this situation. If you would like, you should be able to create a PR from a fork, and I'll be glad to review it.

**Comment by arnounesco** [comment]
Christopher,

I guess I'm doing something wrong but this is what I get when creating a pull request:

Can't create a new pull request: Push failed: ERROR: Permission to cstarcher/pysaf.git denied to arnounesco. Could not read from remote repository. Please make sure you have the correct access rights and the repository exists.

vriendelijke groeten / kin​d regards

Arno Lambert
IT Services Manager
UNESCO/IOC Project Office for IODE
Innov​Ocean Campus - Jacobsenstraat 1 - 8400 Oostende - BELGIUM
Email: ***@***.***> ***@***.***> ***@***.******@***.***>
OceanExpert ID: <http://www.oceanexpert.net/expert/6552> <https://www.oceanexpert.net/expert/35711> https://www.oceanexpert.org/expert/35711
URL: <http://www.iode.org> https://www.iode.org<http://www.iode.org>, https://oceanbestpractices.org, https://oceaninfohub.org, https://odis.org
________________________________
Van: Christopher Starcher ***@***.***>
Verzonden: dinsdag 8 augustus 2023 16:38
Aan: cstarcher/pysaf ***@***.***>
CC: Lambert, Arno ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [cstarcher/pysaf] DMG cannot be run on OS X (Issue #6)

CAUTION: This email is external from UNESCO. Please be vigilant on its sender and content. ATTENTION : Cet e-mail est externe à l'UNESCO. Soyez vigilant sur son expéditeur et contenu.



@arnounesco<https://github.com/arnounesco>

Yes, I've not updated the README for this situation. If you would like, you should be able to create a PR from a fork, and I'll be glad to review it.

**Comment by cstarcher** [comment]
Arno,

I'm not sure what's going wrong. Did you commit to your fork and make a PR from it to the PySAF repository? I just tested this with a separate account and was able to create the PR.
